Use dedicated controllers for mixed attributes (#18879)
dotnet/AspNetCore.Docs: aspnetcore/security/authorization/simple.md

## Authorize attribute and Razor Pages

> The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> can ***not*** be applied to Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. Consider using an ASP.NET Core MVC controller for pages that require mixed authorization attributes on different handlers.

Reviewer: ```suggestion
The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> can ***not*** be applied to Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. Consider using an ASP.NET Core MVC controller for pages with different authorization requirements for different handlers.
```

Reviewer: Or something like that. Your original content read fine to me
